app/models/concerns/attachment_auto_rotate_concern.rb

```diff
@@ -0,0 +1,22 @@
+module AttachmentAutoRotateConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_create_commit :auto_rotate
+  end
+
+  private
+
+  def auto_rotate
+    return if blob.nil?
+    return if ["image/jpeg", "image/jpg"].exclude?(blob.content_type)
+
+    blob.open do |file|
+      Tempfile.create(["rotated", File.extname(file)]) do |output|
+        processed = AutoRotateService.new.process(file, output)
```

Comment:
```suggestion
        processed = AutoRotateService.process(file, output)
```

tu peux peut être en faire des fonctions statiques vu qu'il n'y a pas de var d'instance ?

Par ailleurs, est ce que tu peux éviter de réuploader s'il n'y pas de rotation à faire ?